flow/raster_cache.cc

```diff
@@ -158,27 +158,28 @@ void RasterCache::Prepare(PrerollContext* context,
   entry.access_count = ClampSize(entry.access_count + 1, 0, access_threshold_);
   entry.used_this_frame = true;
   if (!entry.image.is_valid()) {
-    entry.image = Rasterize(context->gr_context, ctm, context->dst_color_space,
-                            checkerboard_images_, layer->paint_bounds(),
-                            [layer, context](SkCanvas* canvas) {
-                              SkISize canvas_size = canvas->getBaseLayerSize();
-                              SkNWayCanvas internal_nodes_canvas(
-                                  canvas_size.width(), canvas_size.height());
-                              internal_nodes_canvas.addCanvas(canvas);
-                              Layer::PaintContext paintContext = {
-                                  (SkCanvas*)&internal_nodes_canvas,
-                                  canvas,
-                                  context->gr_context,
-                                  nullptr,
-                                  context->raster_time,
-                                  context->ui_time,
-                                  context->texture_registry,
-                                  context->raster_cache,
-                                  context->checkerboard_offscreen_layers};
-                              if (layer->needs_painting()) {
-                                layer->Paint(paintContext);
-                              }
-                            });
+    entry.image = Rasterize(
```

Comment:
I guess the formatting of this file is going to depend on the last editor? 😝 
@chinmaygarde 

[not blocking this PR]

Reply:
The actual change here is that `paintContext.rasterCache` is getting set to `context->has_platform_view ? nullptr : context->raster_cache` instead of just `context->raster_cache`. That statement is so long that `clang-format` decided to move this whole block further to the left. I'm not sure why it moved it further to the right to start with, but it should be consistent between editors and CI.